### PR TITLE
Add Motorola Moto G4 (athene)

### PR DESCRIPTION
**Pull request by ChillarAnand**

### CODE REVIEW

**Comment by ChillarAnand** [comment]
Thank you @LNJ2. Added sepolicy repo. 

Documentation didn't mention anything about modifying kernel or  device tree? I am doing this for first time. If you can point out what changes need to be done, that would be helpful.

**Comment by jbruechert** [comment]
I guess modifying the kernel is one of the later steps of the documentation. Once you are at that point and everything builds, we can probably merge this.

**Comment by lnjX** [comment]
@ChillarAnand You need to make modifications to the kernel config, so that a normal GNU/Linux will be able to boot, see the documentation for that: http://docs.halium.org/en/latest/porting/build-sources.html#modify-the-kernel-configuration
When you've done that fork the kernel repository from lineageos, create a new branch `halium-7.1` from the `cm-14.1` branch and commit the new kernel config. Then change the repository in the manifest to your fork. :)

In the device repository, remove all apks and jars from a file called `proprietary-files.txt`.

File: manifests/motorola_athene.xml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+  <remote name="github" fetch="ssh://git@github.com/" />
   <project path="device/motorola/athene" name="android_device_motorola_athene" remote="los" />
   <project path="device/motorola/qcom-common" name="android_device_motorola_qcom-common" remote="los" />
   <project path="device/qcom/common" name="android_device_qcom_common" remote="los" />
   <project path="device/qcom/sepolicy" name="android_device_qcom_sepolicy" remote="los" />
   <project path="external/bson" name="android_external_bson" remote="los" />
   <project path="external/stlport" name="android_external_stlport" remote="los" />
   <project path="external/sony/boringssl-compat" name="android_external_sony_boringssl-compat" remote="los" />
-  <project path="kernel/motorola/msm8952" name="android_kernel_motorola_msm8952" remote="los" />
+  <project path="kernel/motorola/msm8952" name="chillaranand/android_kernel_motorola_msm8952" remote="github" revision="halium-7.1" />
```
Comment (jbruechert) [inline manifests/motorola_athene.xml]: Please just use the `hal` remote, it already sets the revision to halium-7.1, and you can use your own github account with it

File: manifests/motorola_athene.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="github" fetch="ssh://git@github.com/" />
```
Comment (lnjX) [inline manifests/motorola_athene.xml]: You don't use the remote, please remove it.

**Comment by lnjX** [comment]
I saw you edited the proprietary-files, but please also fix this error
```
device/motorola/athene/cmactions/Android.mk:37: frameworks/base/packages/SettingsLib/common.mk: No such file or directory
```
in your device tree

You can just uncoment the line (device/motorola/athene/cmactions/Android.mk:37) or disable the whole cmactions in the main Android.mk.

**Comment by Herrie82** [comment]
Seeing https://github.com/Halium/halium-devices/pull/76 was merged, this one can be closed?

**Pull request closed without merging**